source/guides/token-management/index.markdown

```diff
@@ -76,7 +76,7 @@ By default, an `Application` has a the following TTLs:
 
 To get these values, you can query the `OAuth Policy` for an Application:
 
-{% codetab id:get-oauth-policy langs:curl node php%}
+{% codetab id:get-oauth-policy langs:curl node php java%}
```

Comment:
In the other guides, Java is always the first one. Can you put all the Java snippets in front of `curl` please?